ci/_

```diff
@@ -5,7 +5,13 @@
 # |source| me
 #
 
+base_dir=$(realpath --strip "$(dirname "$0")/..")
+
 _() {
-  echo "--- $*"
+  if [[ $(pwd) = $base_dir ]]; then
+    echo "--- $*"
+  else
+    echo "--- $* (wd: $(pwd))"
```

Comment:
Because of lack of this, I modified [a wrong place](https://github.com/solana-labs/solana/pull/9057/files#diff-4b5263e29d55962afaf13a723bb3d2b9R25).